list/README.md

```diff
@@ -19,14 +19,206 @@ but already use ready-made mechanisms to handle lists in Python.
+```
+[1, 2, 3, 4, 4, 5, 6, 7, 8]
+```
+In cases where we do not want repetition after merging lists, this method will unfortunately not work.
+
+Since Python 3.5, you can alternatively concatenate lists using [*list1 , *list2]. 
+```
+list1 = [1,2,3,4]
+list2 = [5,6,7,8]
+
+merged_list = [*list1, *list2]
+print(merged_list)
+```
+Unfortunately, this way is also not suitable for merging lists without repetition. 
+
+We can use the extend() function to merge lists:
+```
+list1 = [1,2,3,4]
+list2 = [5,6,7,8]
+
+merged_list = []
+merged_list.extend(list1)
+merged_list.extend(list2)
+
+print(merged_list)
+```
+Of course, it is also unsuitable in case you do not want to have duplicates.
+
+Now I will do the old way of merging lists with each other, so that there is no repetition. 
+```
+list1 = [1,2,3,4]
+list2 = [4,5,6,7,8]
+
+merged_list = list1
+
+for item in list2:
+    if item not in merged_list:
+        merged_list.append(item)
+
+print(merged_list)
+```
+And a nicer way to merge lists:
+```
+list1 = [1,2,3,4]
+list2 = [4,5,6,7,8]
+
+merged_list = list(set(list1 + list2))
+print(merged_list)
+```
+The set function has been with us since python 2 but unfortunately it is rarely seen in use, and it can really simplify list merging for us when we don't want to have repetition. 
+
 ## The Filter() Function
 
+The filter() function, as the name suggests, is used to filter data in lists. 
+The feature has been with us since Python version 2, but many people continue not to use it.
+
+Below is an example without using the filter function:
+```
+list1 = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+
+filtered_list = []
+
+for item in list1:
+    if item % 2 == 0:
+        filtered_list.append(item)
+        
+print(filtered_list)
+```
+Output:
+```
+[2, 4, 6, 8, 10]
+```
+
+Now code with use the filter function:
+```
+list1 = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+
+# returns True if number is even
+def check_even(number):
+    if number % 2 == 0:
+          return True  
+    return False
+        
+filtered_list = list(filter(check_even, list1))
+print(filtered_list)
+```
+Output is the same. 
+
+The filter function can be applied not only to numbers, but it can filter anything we want from the list. Below is the code to check if filtering by the letter b:
+```
+list1 = ['beaver', 'lemon', 'beard', 'cat', 'dog']
+
+def check_even(word):
+    if 'b' in word:
+          return True  
+    return False
+        
+filtered_list = list(filter(check_even, list1))
+print(filtered_list)
+```
+Output:
+```
+['beaver', 'beard']
+```
+
 ## List Comprehensions
 
 ## The Map() Function
 
 ## The Zip() functions
 
+The Zip() function in Python serves to create Tuple object. The Zip() function as paremteter use Lists, strings, numbers, tuple, etc. 
+The Zip() function returns an iterator of tuples, where the i-th tuple contains the i-th element from each of the argument sequences or iterables.
+The following example, using the zip(0) function, creates a tuple object from two lists:
```

Comment:
zip(0) ? skad to zero :)

Reply:
nie miało byc zero, za szybko pisalem :+1: